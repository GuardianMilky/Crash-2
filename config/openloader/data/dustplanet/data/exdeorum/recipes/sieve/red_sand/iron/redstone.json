{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:red_sand"},"mesh":"exdeorum:iron_mesh","result":"minecraft:redstone","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.11}}