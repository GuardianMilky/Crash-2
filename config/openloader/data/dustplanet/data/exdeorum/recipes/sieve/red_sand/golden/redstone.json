{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:red_sand"},"mesh":"exdeorum:golden_mesh","result":"minecraft:redstone","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.07}}